frontend/public/locales/en/translation.json

```diff
@@ -1338,7 +1338,7 @@
     "Image Content Sources": "Image Content Sources",
     "Image registry mirror": "Image registry mirror",
     "import": "Import",
-    "import.auto.button": "Import",
+    "Import cluster YAML": "Import cluster YAML",
```

Comment:
Where is "Import cluster YAML"? Does that give enough information?

Reply:
This appears with the YAML editor... not something new, just a string that was missing from the file.
<img width="1976" alt="image" src="https://user-images.githubusercontent.com/42188127/180804685-a0e1d6bd-a36b-49c1-a2b1-f9da7e58d520.png">